client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import scss from './styles.scss';
```

Comment:
I thought that (since we are hopefully going to use PostCSS at some point) it's easier to spot files that need updating when they are using correct file extension.